CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `partiql_ast_passes::static_typer` for type annotating the AST.
```

Comment:
I will remove this line, after we decide to remove the current `ast` pass or not.

Reply:
I am hesitating to remove it just now, but can add a `TODO` to revisit; we should just make a final decision very soon.